docs/docs/troubleshooting/LIMITATIONS.md

```diff
@@ -0,0 +1,5 @@
+# Limitations 
+We haven't changed our Data Center applications' architecture to support Kubernetes. So, as is with all our Data Center products, the following limitiations still exist:
+* We don’t support horizontal or vertical autoscaling in our products. Read about [Product scaling](../userguide/resource_management/RESOURCE_SCALING.md).
```

Comment:
I can see that some contractions use the apostrophe while "don't" on this line has a quotation mark (a Unicode symbol). Can you change them to be consistent? For Markdown documents, I would prefer the regular ASCII apostrophe (you might have to disable the "smart quote" feature.) I'll leave it up to you.